api/districts.ts

```diff
@@ -23,13 +24,14 @@ export default async (req: NowRequest, res: NowResponse) => {
     res.setHeader('Cache-Control', 's-maxage=3600');
 
     let districts = [];
-    
-    const repsonse = await axios.get("https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/RKI_Landkreisdaten/FeatureServer/0/query?where=1%3D1&outFields=GEN,cases,deaths,cases_per_100k,cases_per_population,last_update,cases7_per_100k&outSR=4326&f=json");
-    const apidata: APIData = repsonse.data;
+
+    const response = await axios.get("https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/RKI_Landkreisdaten/FeatureServer/0/query?where=1%3D1&outFields=GEN,county,cases,deaths,cases_per_100k,cases_per_population,last_update,cases7_per_100k&outSR=4326&f=json");
+    const apidata: APIData = response.data;
 
     for (const feature of apidata.features) {
         let district = new District();
         district.name = feature.attributes.GEN;
+        district.type = feature.attributes.county.substring(0,2);
```

Comment:
Using `.substring(0,2)` does not always returns a meaningful result. For example: `Region Hannover` results in `st`.

We should remove `.substring(0,2)` and return the whole information and let the client decide what to do with this. information

Reply:
Sure, why not. Just thinking whether it might be helpful to add the state name as well (the ``BL`` field). What do you think?